source/tutorials/installing-nixos-on-a-raspberry-pi.md

```diff
@@ -42,6 +42,8 @@ $ nix-shell -p wget zstd
 :::{note}
 You can pick a newer image by going to [Hydra job](https://hydra.nixos.org/job/nixos/trunk-combined/nixos.sd_image.aarch64-linux),
 clicking on a build and copying the link to the build product image.
+There have been reported [some problems](https://github.com/NixOS/nixpkgs/issues/179701) with the latest sd images on the pi 4, where it turns of the display while booting up.
```

Comment:
```suggestion

There have been reported [some problems](https://github.com/NixOS/nixpkgs/issues/179701) with the latest SD images on the Pi 4, where it turns off the display while booting up.
```

Open a new paragraph for each topic. Lumping everything together makes the text harder to scan and parse, and most readers don't actually read.

Please run basic spell check before submitting PRs. Use the official spelling of product names.